lib/private/strip-stack-message.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function stripStackMessage(e) {
```

Comment:
`getBareTraceFromError` will definitely not remind me of what it's doing...I can change it to `stripMessageFromStackTrace` or `getStackTraceWithoutMessage`?